Client/script_index.js

```diff
@@ -0,0 +1,51 @@
+let button = document.getElementById("submitBtn");
+let userInput = document.getElementById("searchInput");
+let luckyButton = document.getElementById("luckyBtn");
+userInput.addEventListener('input', buttonEnable);
+button.addEventListener('click', storeSearch);
+luckyButton.addEventListener('click', e => {getFirstResult(e)});
+
+ //const KEY = YOUR KEY GOES HERE
+ //const CX = YOUR CX CODE GOES HERE
+
+
+function buttonEnable(){
+    if (userInput.value !== ""){
+        button.disabled = false; 
+        luckyButton.disabled = false; 
+    } else {
+        button.disabled = true;
+        luckyButton.disabled = true;
+    }
+}
+
+function storeSearch(){
+    let userInputValue = userInput.value;
+    localStorage.setItem("searchInput", userInputValue);
+}
+
+async function getFirstResult(e){
+    e.preventDefault();
+    let userInputValue = userInput.value;
+    let data = await fetch(`http://localhost:3000/${userInputValue}`)
```

Comment:
When i tried adding inputs to the search bar as instructed on the README, my dev tools had shown me errors that it had failed to fetch any data from the localhost, not too sure as to why that was happening, but the buttons didnt work till i typed an input in so thats good, shows its functioning as you had intended it too.